torch/utils/_stats.py

```diff
@@ -3,17 +3,23 @@
 # AND SCRUB AWAY TORCH NOTIONS THERE.
 import collections
 import functools
-from typing import Any, Callable, OrderedDict
+from typing import Callable, OrderedDict, TypeVar
+from typing_extensions import ParamSpec
+
 
 simple_call_counter: OrderedDict[str, int] = collections.OrderedDict()
 
+_P = ParamSpec("_P")
+_R = TypeVar("_R")
+
+
 def count_label(label: str) -> None:
     prev = simple_call_counter.setdefault(label, 0)
     simple_call_counter[label] = prev + 1
 
-def count(fn: Callable[..., Any]) -> Callable[..., Any]:
+def count(fn: Callable[_P, _R]) -> Callable[_P, _R]:
```

Comment:
This is unearthing a real error, but that doesn't need to be fixed in this PR. Feel free to and a type ignore at the error below.

Reply:
Got it! Added that and a fix to a lint error about trailing spaces that came up after the ignore comment.